frontend/src/components/common/FridgeCheckInDescription.tsx

```diff
@@ -0,0 +1,67 @@
+import { EditIcon } from "@chakra-ui/icons";
+import { Button, Link, Text } from "@chakra-ui/react";
+import React from "react";
+
+export const FridgeCheckInDescription = () => (
+  <>
```

Comment:
Placeholder for now, can use the content API client at the very end, just because a lot of people probably haven't seeded their local content table